src/core/src/main/java/org/apache/jmeter/gui/action/KeyStrokes.java

```diff
@@ -72,6 +72,7 @@ private static int getMenuShortcutKeyMask() {
     public static final KeyStroke OPEN              = KeyStroke.getKeyStroke(KeyEvent.VK_O, CONTROL_MASK);
     public static final KeyStroke EXIT              = KeyStroke.getKeyStroke(KeyEvent.VK_Q, CONTROL_MASK);
     public static final KeyStroke ACTION_START      = KeyStroke.getKeyStroke(KeyEvent.VK_R, CONTROL_MASK);
+    public static final KeyStroke ACTION_START_NO_PAUSE  = KeyStroke.getKeyStroke(KeyEvent.VK_N, CONTROL_MASK | InputEvent.SHIFT_DOWN_MASK);
```

Comment:
`Ctrl`+`Shift`+`N` feels a bit like a command to create something *new*. But as the menu shortcut uses `n` already, this seems to be a natural choice here.